package.json

```diff
@@ -11,6 +11,7 @@
     "bootstrap": "^5.1.0",
     "cors": "^2.8.5",
     "dotenv": "^10.0.0",
+    "jsonwebtoken": "^8.5.1",
```

Comment:
You shouldn't need `jsonwebtoken` on the front-end. That should all be handled by `@auth0/auth0-react`.